facets_mcp/tools/deploy_module.py

```
@@ -19,12 +19,25 @@ def list_test_projects() -> str:
-        str: List of all test projects
+        str: A JSON string success message and list of all test projects or error message
     """
     api_instance = UiStackControllerApi(ClientUtils.get_client())
     stacks = api_instance.get_stacks_using_get1()
     stack_names = [stack.name for stack in stacks if stack.preview_modules_allowed]
-    return stack_names if stack_names else "No test projects found. Ask the user to create one from the Facets UI."
+    if stack_names:
+        return json.dumps({
+            "success": True,
+            "message": "Succesfully retrieved the names of all available test projects.",
+            "instructions": f"Inform User: Choose a test project from the project list.",
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)